test/support/test_helpers.ex

```diff
@@ -1,8 +1,10 @@
 defmodule ResearchResource.TestHelpers do
-  alias ResearchResource.Repo
+  alias ResearchResource.{Repo, User}
+  alias Plug.Conn
```

Comment:
Ok thanks for the changes on the PR :+1: I'm not sure also if there are any issues using Plug.Conn instead of ConnCase in tests, I will open another issue and try to investigate the details of this two libraries